README.md

```diff
@@ -186,7 +187,7 @@ zone.  This role has a trust relationship with the users account.
 | sessendemail\_role\_name | The name to assign the IAM role (as well as the corresponding policy) that allows sufficient permissions to send email via AWS SES. | `string` | `"SesSendEmail-cyber.dhs.gov"` | no |
 | tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
 
-## Outputs ##
+## Outputs
```

Comment:
Revert this change that terraform-docs makes:
```suggestion
## Outputs ##
```

Reply:
Fixed in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/103/commits/5de085cd1da6a42095891d88144f4bf0b14543a4.